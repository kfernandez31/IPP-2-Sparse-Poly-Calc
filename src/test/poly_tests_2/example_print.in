0
PRINT
1
PRINT
-2
PRINT
(0,0)
PRINT
(1,0)
PRINT
(-2,0)
PRINT
(1,1)
PRINT
(1,0)+(1,2)
PRINT
(1,2)+(1,0)
PRINT
(1,2)+(-1,2)
PRINT
(1,2)+(-2,2)
PRINT
((1,2),15)+(-7,8)
PRINT
(3,1)+(((4,4),100),2)
PRINT
