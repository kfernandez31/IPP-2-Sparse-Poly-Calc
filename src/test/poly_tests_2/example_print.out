0
1
-2
0
1
-2
(1,1)
(1,0)+(1,2)
(1,0)+(1,2)
0
(-1,2)
(-7,8)+((1,2),15)
(3,1)+(((4,4),100),2)
